tests/test_torch_frontend.py

```diff
@@ -0,0 +1,273 @@
+"""
+tests for returnn.torch.frontend
+"""
+
+import _setup_test_env  # noqa
+
+import torch
+import pytest
+
+from returnn.tensor import Tensor, Dim, batch_dim
```

Comment:
`batch_dim` should not be used here. Rather, just create your own `Dim(...)` for it.

Reply:
Note: It's not really so clear yet whether we need the `batch_dim` at all for the new frontend API. I think not. Maybe it should be moved elsewhere, or only specific for the old RETURNN net dict. Not sure.

Related is also `single_step_dim`. I didn't really thought about it yet.

But just avoid using it, unless there is a good reason, but then let's discuss first.